guestbook.py

```diff
@@ -33,14 +33,22 @@ def guestbook_key(guestbook_name=DEFAULT_GUESTBOOK_NAME):
     """Constructs a Datastore key for a Guestbook entity with guestbook_name."""
     return ndb.Key('Guestbook', guestbook_name)
 
+
 # [START greeting]
+class Author(ndb.Model):
+    """Sub model for representing an author."""
+    id = ndb.StringProperty(indexed=False)
```

Comment:
There may be confusion since id is also used for "key id". However this is something that is explicitly allowed by ndb. Up to you if you want to rename or not.